content/learn/book/faq/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Faq"
+title = "FAQ"
```

Comment:
sneaky to include in this pr, but I'm on board😂

Reply:
Heh, I hope no one else minds. I thought it was some sort of formatting issue, but figured all caps made sense.